system_test/helpers/aest_nodes.erl

```diff
@@ -16,7 +16,7 @@
 -export([stop_node/3]).
 -export([kill_node/2]).
 -export([extract_archive/4]).
--export([run_cmd_in_node_dir/3]).
```

Comment:
Can we keep this function with default timeout? I see the default (5000) timeout is used quite often.

Reply:
addressed

Reply:
Seconded, it would remove all changes for lines with the argument `5000` and make the other differences to test cases stand out more.